src/test/ui/consts/uninhabited-const-issue-61744.rs

```diff
@@ -1,15 +1,15 @@
 // build-fail
 
 pub const unsafe fn fake_type<T>() -> T {
-    hint_unreachable()
+    hint_unreachable() //~ 4:23: any use of this value will cause an error [const_err]
```

Comment:
I think you need to replace the `4:23:` with `ERROR`, but I may be missing something new that changed in the ui test suite